app/controllers/organizations/adopter_fosterer/fostered_pets_controller.rb

```diff
@@ -0,0 +1,18 @@
+module Organizations
+  module AdopterFosterer
+    class FosteredPetsController < Organizations::BaseController
+      before_action :context_authorize!
+      layout "adopter_foster_dashboard"
+
+      def index
```

Comment:
So this will currently return all of my past, present, and future fosters, right?
I think for now we should just show fosters that are current (there is a scope for this on Match). In the future, if orgs want us to show more, we can handle this with filters for the user to select past, current, or future fosters.

Reply:
Thank you for the details.. I overlooked details related to past and future fosters. Will update query accordingly to show only current fosters

Reply:
Done